packages/web-app-files/src/components/Upload/DetailsWidget.vue

```diff
@@ -5,8 +5,12 @@
         <oc-icon name="file_copy" class="oc-mr-s" />
         <div class="uk-width-expand">
           <div class="uk-flex">
-            <div class="oc-text-bold uk-width-expand uk-text-truncate">{{ item.name }}</div>
-            <div class="uk-width-auto uk-text-nowrap">{{ getResourceSize(item.size) }}</div>
+            <div id="item-name" class="oc-text-bold uk-width-expand uk-text-truncate">
+              {{ item.name }}
+            </div>
+            <div id="item-size" class="uk-width-auto uk-text-nowrap">
```

Comment:
Same as for the other `id` property some lines above. `files-upload-details-item-size` would be more appropriate.